app/models/mixins/miq_web_server_worker_mixin.rb

```diff
@@ -105,12 +105,12 @@ def self.sync_workers
       result
     end
 
-    def self.thin_pid_file(port)
-      Rails.root.join("tmp/pids/thin.#{port}.pid")
+    def self.pid_file(port)
+      Rails.root.join("tmp/pids/puma.#{port}.pid")
```

Comment:
I prefer keeping it generic too. This way we can more easily switch to something else in the future.